app/controllers/staff/activity_forms_controller.rb

```diff
@@ -211,6 +214,10 @@ def recipient_country
     params.require(:activity).permit(:recipient_country).fetch("recipient_country", nil)
   end
 
+  def intended_beneficiaries
+    params.require(:activity).permit(intended_beneficiaries: []).fetch("intended_beneficiaries", nil)
```

Comment:
I used the `drop(1)` because I was getting an empty string as first item of the array `intended_beneficiaries`, regardless of having other countries selected, and this was giving me problems with views and specs. I thought that empty string was generated by the default empty hidden field on the checkbox fieldset. Am I right? I've not used govUK checkboxes before

Reply:
This might be an artifact of how Rails does checkboxes and radio buttons -- it generates something like this so the browser sends `0` if the checkbox is unchecked and `1` if it is set:

```html
<input type="hidden" name="model[field]" value="0">
<input type="checkbox" name="model[field]" value="1">
```

I'm not sure how this applies on these checkbox lists though -- how do I get a page in development where I can see this working?